src/components/GameCard.tsx

```diff
@@ -75,10 +100,13 @@ export function GameCard(gameCardProps : GameCardProps ) {
             </div>
             <p>{gameCardProps?.dateAdded?.toDateString()}</p>
             <div>
-                <OpenLibraryModalButton text="Add to Library" onClick={addToLibrary}></OpenLibraryModalButton>
+                <Select
+                    placeholder="Select a Library"
+                    options={libraryOptions}
+                    onChange={(library) => setSelectedLibrary(library!!.value)}
+                />
+                <OpenLibraryModalButton text="Add to Library" onClick={() => addToLibrary(gameCardProps.gameId, selectedLibrary)}></OpenLibraryModalButton>
```

Comment:
We should rename this component since it's not actually opening a modal.

Also, it looks like this doesn't have any children so we can make it a self-closing component - you can do this instead
```suggestion
                <OpenLibraryModalButton text="Add to Library" onClick={() => addToLibrary(gameCardProps.gameId, selectedLibrary)}/>
```

Reply:
Yeah good point! I guess it depends on how we'll want it to look in the long term (I thought about adding the select logic in there, but it was easier to reason about this way). But renaming back and forth is easy so I can do that

Reply:
and re: self-closing component: yes! rude of typescript to autocorrect it for me the other way, switching it